documentation/docs/guide/GettingStarted/deploy_openshift.md

```diff
@@ -0,0 +1,128 @@
+# Deploying to OpenShift
```

Comment:
These seem slightly redundant, given they are almost the same instructions

Reply:
Yes i was thinking about a way to maybe to combine, but there are some distinct difference, one the base file (super important) and the command, although `oc == kubectl` for 98% of it

Reply:
@kshapka-bcgov welcome any edits or suggestions, to combine them, as long as it doesn't confuse the reader.